fem/fe/fe_base.hpp

```diff
@@ -360,7 +368,8 @@ class FiniteElement
    /** @brief Evaluate the values of all shape functions of a scalar finite
        element in physical space at the point described by @a Trans. */
    /** The size (#dof) of the result Vector @a shape must be set in advance. */
-   void CalcPhysShape(ElementTransformation &Trans, Vector &shape) const;
+   virtual void CalcPhysShape(ElementTransformation &Trans,
+                              Vector &shape) const;
```

Comment:
Why make this method virtual? To me it looks like it does not need to be.

Reply:
I don't think it makes a difference for this PR. I can change it back. I probably just didn't see a reason why it shouldn't be virtual.